### PR TITLE
Apply correct formatting to comments in the Vulkan driver

### PR DESCRIPTION
**Pull request by RandomShaper**

### CODE REVIEW

File: drivers/vulkan/rendering_device_vulkan.cpp [hunk 1]
```diff
@@ -860,7 +860,7 @@ uint32_t RenderingDeviceVulkan::get_image_format_pixel_size(DataFormat p_format)
 		case DATA_FORMAT_D24_UNORM_S8_UINT:
 			return 4;
 		case DATA_FORMAT_D32_SFLOAT_S8_UINT:
-			return 5; //?
+			return 5; // ?.
```
Comment (aaronfranke) [inline drivers/vulkan/rendering_device_vulkan.cpp:863]: This comment can probably just be removed...?

File: drivers/vulkan/rendering_device_vulkan.cpp [hunk 2]
```diff
@@ -6748,19 +6748,19 @@ RID RenderingDeviceVulkan::render_pipeline_create(RID p_shader, FramebufferForma
 	VkPipelineFragmentShadingRateStateCreateInfoKHR vrs_create_info;
 	if (context->get_vrs_capabilities().attachment_vrs_supported) {
 		// If VRS is used, this defines how the different VRS types are combined.
-		// combinerOps[0] decides how we use the output of pipeline and primitive (drawcall) VRS
-		// combinerOps[1] decides how we use the output of combinerOps[0] and our attachment VRS
+		// CombinerOps[0] decides how we use the output of pipeline and primitive (drawcall) VRS.
+		// CombinerOps[1] decides how we use the output of combinerOps[0] and our attachment VRS.
```
Comment (aaronfranke) [inline drivers/vulkan/rendering_device_vulkan.cpp:6752]: This looks like a piece of code, so it must match the casing of that code.

Comment (RandomShaper) [reply drivers/vulkan/rendering_device_vulkan.cpp]: Fixed!

File: drivers/vulkan/rendering_device_vulkan.cpp (hunk 1 above)
Comment (RandomShaper) [reply drivers/vulkan/rendering_device_vulkan.cpp]: I believe the comment has value (it's not clear whether that's the right value to return in that case). However, a period is redundant after a question mark, so I've kept the question mark alone.

**Review by aaronfranke — APPROVED** [review]

**Comment by akien-mga** [comment]
Thanks!

**Pull request merged**